code/src/components/EndScreen.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+import { useSelector } from "react-redux";
+import styled from "styled-components/macro";
+import img from "../assets/7.jpg";
```

Comment:
remember to separate different types of input from each other with an empty line.